src/quo2/components/numbered_keyboard/keyboard_key/style.cljs

```diff
@@ -0,0 +1,19 @@
+(ns quo2.components.numbered-keyboard.keyboard-key.style)
+
+(def container
+  {:width           48
+   :height          48
+   :justify-content :center
+   :align-items     :center
+   :border-radius   999})
+
+(defn text
+  [color]
+  {:color     color
+   :font-size 27})
```

Comment:
these font sizes are all set by the quo2 text component, in quo2.components.markdown.text

Reply:
Done